frontend/src/views/WheresMyGene/common/store/actions.ts

```diff
@@ -70,6 +70,15 @@ export function selectTissues(
   };
 }
 
+export function setFilterDimensions(
```

Comment:
added filterDimensions to the reducer in order to keep track of the latest filterDimensions set when manipulating the tissue QuickSelect.

this is to dodge a bug where hitting a tissue will cause the filters to return empty as "isLoading" will be true and transiently return null for useWMGQueryRequestBody.

so, if we get an empty filter object after hitting a tissue, we default to the previous filters before hitting the tissue until filters are all properly updated.

convoluted!!!!